### PR TITLE
Hackathon writeup

### PR DESCRIPTION
**Pull request by Willow-Systems**
- Add hackathon post
- Update /hackathon-001

### CODE REVIEW

**Comment by ishotjr** [comment]
awesome! will review tonight!! 💙✨

**Review by crc-32 — CHANGES_REQUESTED** [review]
just felt stuff was a little easier to read like this
❤️

**Review by ishotjr — CHANGES_REQUESTED** [review]
Fantastic write-up! Couple of typos etc. when you have a moment, otherwise, LGTM! :bowtie: 🚀

File: _posts/2023-05-12-a-look-back-at-the-rebble-hackathon.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+## Watch apps
+
+We decided to break the watch apps category in half for picking winners: Tools and Games. We received a wide number of tools, but before we get to the winners, let's look at all the watch apps across both categories that were submitted!
+
+![](/images/hackathon-look-back/apps.png)
+
+The quality of apps submitted during the hackathon blew me away. Watchapps can be tricky to make, and take a lot of time; it cannot be overstated just how impressive _all_ the submissions were. From top-left to bottom-right the submissions are:
+
+- [Services for Pebble by Blockarchitect - Manage your planning center services on your wrist](https://apps.rebble.io/en_US/application/6377f2c7fdf3e30009f63951)
```
Comment (blockarchitech) [inline _posts/2023-05-12-a-look-back-at-the-rebble-hackathon.md:114]: My name is slightly wrong here 😅

It's supposed to be a play on words thing but turns out it just makes it impossible for other people to type ha

**Review by blockarchitech — CHANGES_REQUESTED** [review]
Looks awesome! Just this I found

File: _posts/2023-05-12-a-look-back-at-the-rebble-hackathon.md (hunk 1 above)
Comment (Willow-Systems) [reply _posts/2023-05-12-a-look-back-at-the-rebble-hackathon.md]: I so nearly got through it all without misspelling anyone's name 😁

**Comment by Willow-Systems** [comment]
@jwise I'm impatient so I'm going to push now 😁 Feel free to make any changes still!

**Pull request merged**